sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileDataSourceV2.scala

```diff
@@ -49,6 +51,13 @@ trait FileDataSourceV2 extends TableProvider with DataSourceRegister {
   }
 
   protected def getTableName(paths: Seq[String]): String = {
-    shortName() + ":" + paths.mkString(";")
+    val name = shortName() + " " + paths.map(qualifiedPathName).mkString(",")
```

Comment:
Sorry, it seems that I missed this last night. `","` -> `";"`?

Reply:
Oh, this is following the output of V1. I am actually fine with either way. WDYT?

Reply:
Maybe, I missed the context. Which part are you referring? It seems that we replaced ":" to " ", too. Are both of them changed to be consistent with the output of V1?

Reply:
Yes, I think so.
We have to change the ":" to " ", otherwise the output will be like "orc:file:/path.."